src/client/Microsoft.Identity.Client/Cache/Items/MsalAccessTokenCacheItem.cs

```diff
@@ -240,5 +240,10 @@ internal bool NeedsRefresh()
             return RefreshOn.HasValue &&
                 RefreshOn.Value < DateTime.UtcNow;
         }
+
+        internal bool IsExpired()
+        {
+            return ExpiresOn < DateTime.UtcNow + Constants.AccessTokenExpirationBuffer;

```

Comment:
why is this buffer added now?

Reply:
Both ADAL and MSAL add 5 min to expiration. This is not new. We just moved the code here, since computing expiration is done in tons of places now.

Reply:
i see. thanks